yarn.config.cjs

```diff
@@ -96,11 +96,19 @@ module.exports = defineConfig({
           // exports correctly.
           expectCorrectWorkspaceExports(workspace);
           // All non-root packages must have the same "build" script.
-          expectWorkspaceField(
-            workspace,
-            'scripts.build',
-            'ts-bridge --project tsconfig.build.json --verbose --clean --no-references',
-          );
+          if (workspace.ident !== '@metamask/design-tokens') {
+            expectWorkspaceField(
+              workspace,
+              'scripts.build',
+              'ts-bridge --project tsconfig.build.json --verbose --clean --no-references',
+            );
+          } else if (workspace.ident === '@metamask/design-tokens') {
+            expectWorkspaceField(
+              workspace,
+              'scripts.build',
+              'ts-bridge --project tsconfig.build.json --verbose --clean --no-references && yarn build:css',
+            );
+          }
```

Comment:
Updating yarn constraints so the build of the css file doesn't get overridden in the design tokens package